MSAL/src/MSAL_Internal.h

```diff
@@ -27,7 +27,7 @@
 
 #define MSAL_VER_HIGH       1
 #define MSAL_VER_LOW        1
-#define MSAL_VER_PATCH      13
+#define MSAL_VER_PATCH      14-SPMTest
```

Comment:
I assume this one is for testing purposes

Reply:
Yes, just for testing. Will clean it up once all testing is done.